ansible_collections/arista/avd/roles/eos_validate_state/tasks/main.yml

```diff
@@ -0,0 +1,281 @@
+---
+
+    - name: include device intended structure configuration variables
+      include_vars: '{{ inventory_dir }}/intended/structured_configs//{{ inventory_hostname }}.yml'
+      delegate_to: localhost
+      tags:
+        - always
+
+    - name: run show version on EOS devices
+      eos_command:
+        commands: "show version | json"
+      register: eos_version
+      tags:
+        - platform_information
+
+    - name: Display device platform and release information
```

Comment:
We should use include per type of validation to make it easier to maintain:

```
- name: Load HW validation
  include_tasks: "hardware.yml"
  tags:
    - hardware
```

Reply:
+1 agreed with this recommendation!